SIL.Core/Reporting/ExceptionHandler.cs

```diff
@@ -37,19 +37,6 @@ public static void Init()
 			_singleton = GetObjectFromSilWindowsForms<ExceptionHandler>() ?? new ConsoleExceptionHandler();
 		}
 
-		/// <summary>
-		/// Initialize the ExceptionHandler.
-		/// </summary>
-		/// <typeparam name="T">Type of exception handler</typeparam>
-		/// <exception cref="InvalidOperationException">If ExceptionHandler.Init has been called before</exception>
-		public static void Init<T>() where T: ExceptionHandler, new()
-		{
-			if (_singleton != null)
-				throw new InvalidOperationException($"An ExceptionHandler (of type ${_singleton.GetType()}) has already been set.");
-
-			_singleton = new T();
-		}
-
 		/// <summary>
 		/// Use this method if you want to use an exception handler besides the default.
```

Comment:
This comment should probably be changed now because it's really the only non-deprecated version.